File: 01_Language/01_Functions/python/is_array.py
# coding: utf-8

def is_array(v):
    return isinstance(v, (tuple, list))

if __name__ == '__main__':
    yes = ['this', 'is', 'an array']
    print(is_array(yes))

    no = 'this is a string'
    print(is_array(no))

File: 01_Language/01_Functions/php/is_array.php
<?php

$yes = array('this', 'is', 'an array');

echo is_array($yes) ? 'Array' : 'not an Array';
echo "\n";

$no = 'this is a string';

echo is_array($no) ? 'Array' : 'not an Array';
echo "\n";

$map = ["a" => 1, "b" => 2];
echo is_array($map) ? "true" : "false";
echo "\n";
